src/mixins/input-management/input-management.ts

```diff
@@ -15,13 +15,18 @@ export interface InputManagementData {
     internalValue: string;
 }
 
+export interface InputManagementFocusable {
+    focusInput(): void;
+}
+
 export enum InputManagementAutocomplete {
     Off = 'off',
     On = 'on'
 }
 @Component
 export class InputManagement extends ModulVue
-    implements InputManagementProps, InputManagementData {
+    implements InputManagementProps, InputManagementData, InputManagementFocusable {
+
     @Prop()
```

Comment:
Est-ce que c'est vraiment pertinent de crée une autre interface ici ?
A ce que je comprend un InputManagement va toujours être un InputManagementFocusable. 

Pour simplifier je ferais qu'une interface qui engloble les Props, data et fonction.

Reply:
Ok. Aussi as-tu regarder la @prop focus du InputManagement , je me demande en quoi la method focusInput() nous permet un fonctionnement qu'on ne pouvait pas faire avec la prop?

Reply:
On m'a dit que ça marchait pas alors j'ai fait le fix qui marcherait... par contre focus aurait du marcher >.<.  Idéalement faudrait utiliser la prop.

Reply:
Ok je ne fais que challenger :D

Reply:
Dans le fond j'vais me garder un post-it pour checker ça.

Reply:
Salut @Mboulianne, je vais tester de mon côté si je suis capable de faire fonctionner le focus dans les formulaires avec la @prop focus. Je vous tiens au courant.

Reply:
Merci!